swap_meet/item.py

```diff
@@ -1,2 +1,13 @@
-class Item:
-    pass
+
+class Item():
+    def __init__(self, category="", condition=0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        item_descriptions = [ "Mint", "Excellent", "Good", "Fair", "Poor:", "Trash"]
+        item_descriptions = item_descriptions[::-1]
```

Comment:
Just like we talked about in class, how could we handle floats here? For example, the item is 3.5.